src/Frontend.hs

```diff
@@ -262,7 +258,6 @@ aulaTesting =
   :<|> batchCreateUsers
   :<|> (PageShow <$> Action.persistent mkRandomPassword)
 
-
```

Comment:
Sorry, I keep tripping that rule. Yes, there is (it mentions haddock sections, but I guess that's what you mean, since haddocks section markup, with stars, from module export lists is not defined nor recongnized inside module implementation, I think).